### PR TITLE
fix: search_table API swagger file bug fix

### PR DESCRIPTION
**Pull request by imnishant**
### Summary of Changes
The swagger file of search_table API did not show the request body in the UI because of which API was inaccessible. In this MR, the swagger file's indentation is fixed and an unwanted property from the request body has been removed.

### Tests
No test case required as it's just a bug fix.

### Documentation
N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Review by sewardgw — APPROVED** [review]
LGTM 👍🏻

**Comment by feng-tao** [comment]
thanks for the contribution! could you fix the DCO?

File: search/search_service/api/swagger_doc/table/search_table_filter.yml [hunk 1]
```diff
@@ -3,36 +3,32 @@ This is used by the frontend API to search table information.
...
-            application/json:
-              schema:
-                $ref: '#/components/schemas/SearchTableResults'
-        500:
-          description: Exception encountered while searching
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponse'
+# Edited: Removed paths, post and summary
```
Comment (dorianj) [inline search/search_service/api/swagger_doc/table/search_table_filter.yml:6]: can you remove this comment? This should just live in commit history

**Review by dorianj — CHANGES_REQUESTED** [review]
1 comment (about the comment). thank you for fixing this!

File: search/search_service/api/swagger_doc/table/search_table_filter.yml (hunk 1 above)
Comment (imnishant) [reply search/search_service/api/swagger_doc/table/search_table_filter.yml]: Done 👍🏻

**Review by dorianj — APPROVED** [review]

**Pull request merged**